segments/hostname.sh

```diff
@@ -3,24 +3,33 @@
 
 TMUX_POWERLINE_SEG_HOSTNAME_FORMAT="${TMUX_POWERLINE_SEG_HOSTNAME_FORMAT:-short}"
 
+# check if user has hostname command
+has_hostname() {
```

Comment:
This method is actaully more general, it checks any program given in the arguments. Thus it could be named `command_exists` or something.